docs/api-references.md

```diff
@@ -0,0 +1,829 @@
+| `metadata` _[ListMeta](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#listmeta-v1-meta)_ | Refer to Kubernetes API documentation for fields of `metadata`. |
+| `items` _[MemberCluster](#membercluster) array_ |  |
+
+
+#### MemberClusterSpec
+
+
+
+MemberClusterSpec defines the desired state of MemberCluster.
+
+_Appears in:_
+- [MemberCluster](#membercluster)
+
+| Field | Description |
+| --- | --- |
+| `identity` _[Subject](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#subject-v1-rbac)_ | The identity used by the member cluster to access the hub cluster. The hub agents deployed on the hub cluster will automatically grant the minimal required permissions to this identity for the member agents deployed on the member cluster to access the hub cluster. |
+| `heartbeatPeriodSeconds` _integer_ | How often (in seconds) for the member cluster to send a heartbeat to the hub cluster. Default: 60 seconds. Min: 1 second. Max: 10 minutes. |
+
+
+#### MemberClusterStatus
+
+
+
+MemberClusterStatus defines the observed status of MemberCluster.
+
+_Appears in:_
+- [MemberCluster](#membercluster)
+
+| Field | Description |
+| --- | --- |
+| `conditions` _[Condition](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#condition-v1-meta) array_ | Conditions is an array of current observed conditions for the member cluster. |
+| `resourceUsage` _[ResourceUsage](#resourceusage)_ | The current observed resource usage of the member cluster. It is copied from the corresponding InternalMemberCluster object. |
+| `agentStatus` _[AgentStatus](#agentstatus) array_ | AgentStatus is an array of current observed status, each corresponding to one member agent running in the member cluster. |
+
+
+#### ResourceUsage
+
+
+
+ResourceUsage contains the observed resource usage of a member cluster.
+
+_Appears in:_
+- [MemberClusterStatus](#memberclusterstatus)
+
+| Field | Description |
+| --- | --- |
+| `capacity` _[ResourceList](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#resourcelist-v1-core)_ | Capacity represents the total resource capacity of all the nodes on a member cluster. |
+| `allocatable` _[ResourceList](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#resourcelist-v1-core)_ | Allocatable represents the total resources of all the nodes on a member cluster that are available for scheduling. |
+| `observationTime` _[Time](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#time-v1-meta)_ | When the resource usage is observed. |
+
+
+
+## placement.kubernetes-fleet.io/v1beta1
+
+
+
+### Resource Types
+- [ClusterResourceBinding](#clusterresourcebinding)
+- [ClusterResourcePlacement](#clusterresourceplacement)
+- [ClusterResourceSnapshot](#clusterresourcesnapshot)
+- [ClusterSchedulingPolicySnapshot](#clusterschedulingpolicysnapshot)
+- [Work](#work)
+- [WorkList](#worklist)
```

Comment:
Can we order this list by the importance of the object like put CRP first followed by Work and the rest more internal fields.